internal/action/command.go

```diff
@@ -697,6 +697,34 @@ func (h *BufPane) QuitCmd(args []string) {
 	h.Quit()
 }
 
+func convertLine(h *BufPane, line string) (int, error) {
+	lineNum, err := strconv.Atoi(line)
+	relative := false
+	if err != nil {
+		return 0, err
+	}
+
+	// Check for relative numbers
+	if strings.Contains(line, "-") || strings.Contains(line, "+") {
+		lineNum = h.Buf.GetActiveCursor().Y + 1 + lineNum
+		relative = true
+	}
+
+	if relative {
+		// Handle wrap-over
+		if lineNum <= 0 {
+			lineNum = h.Buf.LinesNum() + 1 + lineNum
+		} else if lineNum > h.Buf.LinesNum() {
+			lineNum = lineNum - h.Buf.LinesNum()
+		} else {
+			// fits
+		}
```

Comment:
I don't think this wrapping makes sense. I think we should just clamp it between the beginning and the end of file, that's the behavior I would expect as a user.

(Actually this clamping is already done below. So now the behavior is even more confusing: we wrap once, and if it still doesn't fit, we clamp it.)

Reply:
Ok, no problem...I was unsure about the wrap-around too.